pages/pipelines/environment_variables.md.erb

```diff
@@ -553,6 +553,27 @@ You can define environment variables in your jobs in a few ways, depending on th
   <p>Do not print or export secrets in your pipelines. See the <a href="/docs/pipelines/secrets">Secrets</a> documentation for further information and best practices.</p>
 </section>
 
+## Variable interpolation
```

Comment:
Is it worth adding subsitution in here somewhere for folks who use that term. Maybe even an aka in the body text?

Reply:
What about if we had substitution in the heading and then interpolation in the body text? Would that just confuse people, or would it be obvious enough and handy for search?

Reply:
Having a totally different term in heading and body always makes wonder why.
I can also solve this on the Algolia side, by adding a synonym - how about that?

Reply:
Sounds like a plan!